avail-subxt/src/api_dev.rs

```diff
@@ -5251,10 +5251,10 @@ pub mod api {
 			use super::runtime_types;
 			pub struct ConstantsApi;
 			impl ConstantsApi {
-				#[doc = " A fee mulitplier for `Operational` extrinsics to compute \"virtual tip\" to boost their"]
```

Comment:
These comments are autogenerated and taken directly from substrate. Manually changing them won't do anything since it will be replaced the next time we call the build_api.sh script.